behave/features/steps/fala_end_to_end_tests.py

```diff
@@ -0,0 +1,41 @@
+from helper.constants import CLA_FALA_URL
+
+from behave import step
+
+
+@step("I am on the Find a legal aid adviser homepage")
+def step_impl_homepage(context):
+    homepage_url = f"{CLA_FALA_URL}"
+    context.helperfunc.open(homepage_url)
+    title_xpath = context.helperfunc.find_by_xpath("//html/body/div/main/div/div/h1")
+    assert title_xpath is not None
+    strip_text = title_xpath.text.replace("\n", " ")
+    assert strip_text, "Find a legal aid adviser or family mediator"
+
+
+@step('I provide the "{location}" details')
+def step_impl_input_location(context, location):
+    input_id = context.helperfunc.find_by_id("id_postcode")
+    input_id.send_keys(location)
+    input_xpath = context.helperfunc.find_by_xpath(
```

Comment:
You can replace that long Xpath with this, `//input[@id="id_postcode"]` It's because there is only one HTML element on the page with that ID.